Plugin.cs

```diff
@@ -53,6 +55,8 @@ private void LoadConfig(Config conf)
                 Log.Debug("Adding initial blank preset");
                 PluginConfig.Instance.ExtraColorPresetsV2.Add(new ExtraColorPresetV2());
             }
+
+            ExtraColorPresetsUniqueSelectable = PluginConfig.Instance.ExtraColorPresetsV2.GetRange(0, PluginConfig.Instance.ExtraColorPresetsV2.Count);
```

Comment:
I think copying the entire preset data is a bad idea, since the presets could be changed in the settings between maps and the changes might not be reflected. I would prefer if the index numbers were stored and picked instea,d that would keep the information DRY.